app/config.py

```diff
@@ -291,12 +272,12 @@ class Config(object):
             'create-nightly-billing': {
                 'task': 'create-nightly-billing',
                 'schedule': crontab(hour=0, minute=15),
-                'options': {'queue': QueueNames.REPORTING},
+                'options': {'queue': QueueNames.NOTIFY},
             },
             'create-nightly-notification-status': {
                 'task': 'create-nightly-notification-status',
                 'schedule': crontab(hour=0, minute=30),
-                'options': {'queue': QueueNames.REPORTING},
```

Comment:
The `PERIODIC` queue seems more appropriate for these since they run once per day. Not something that needs changed, but just thought I'd point it out.

Reply:
I almost decided to put them all in one queue because we're not even using the queues appropriately. I'm planning on making a new ticket to overhaul the queues and workers.